cart/Changelog.md

```diff
@@ -53,4 +53,13 @@
   * New template function to get if any of the currently stored adjustments caused a coupon code to be removed
 
 # 17. January 2020
-* Add `AppliedGiftCard` convenience function `Total()`
\ No newline at end of file
+* Add `AppliedGiftCard` convenience function `Total()`
+
+# 6. February 2020
+* Breaking Change to `EventPublisher` interface, `PublishChangedQtyInCartEvent` and `PublishAddToCartEvent` now
+include a cart as a parameter
+* Breaking Change to behaviour of `AddToCartEvent` and `ChangedQtyInCartEvent`, they are now thrown after
+the cart has been adjusted and written back to cache
```

Comment:
Please mention the new dispatching order of events, first behaviour events then the ones of cartService.